src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2532,17 +2532,17 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 /* Health Certificate Information */
 
-"HealthCertificate_Info_Title" = "Impfzertifikat hinzufügen";
+"HealthCertificate_Info_Title" = "Zertifikate hinzufügen";
 
 "HealthCertificate_Info_imageDescription" = "Eine geschützte Person schaut auf ein Smartphone";
 
-"HealthCertificate_Info_description" = "Fügen Sie Ihren digitalen Impfnachweis in der App hinzu. Sobald Ihr Impfschutz vollständig ist, können Sie den QR-Code in der App vorzeigen, um Ihren Impfschutz nachzuweisen.";
+"HealthCertificate_Info_description" = "Fügen Sie ihre digitalen COVID-Zertifikate in der App hinzu, um mit der App Ihren Impfschutz oder ein negatives Testergebnis nachweisen zu können.";
```

Comment:
```suggestion
"HealthCertificate_Info_description" = "Fügen Sie ihre digitalen EU COVID-Zertifikate in der App hinzu, um mit der App Ihren Impfschutz oder ein negatives Testergebnis nachweisen zu können.";
```
correct name: digitales EU COVID-Zertifikat

Reply:
Hi. Thanks for the updated texts. Just changed it.